Python Simulator/testLoad.py

```diff
@@ -0,0 +1,20 @@
+from eventgame import EventGame
+from player import Player
+
+if __name__ == '__main__':
+  print("Welcome to Square Game")
```

Comment:
What is the purpose of this in a test? You are already using a testing framework.